addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -520,16 +520,20 @@ def _update_accrual(self):
     ####################################################
 
     def onchange(self, values, field_name, field_onchange):
-        # Try to force the leave_type name_get when creating new records
-        # This is called right after pressing create and returns the name_get for
+        # Try to force the leave_type display_name when creating new records
+        # This is called right after pressing create and returns the display_name for
         # most fields in the view.
         if field_onchange.get('employee_id') and 'employee_id' not in self._context and values:
             employee_id = get_employee_from_context(values, self._context, self.env.user.employee_id.id)
             self = self.with_context(employee_id=employee_id)
         return super().onchange(values, field_name, field_onchange)
 
-    def name_get(self):
-        res = []
+    @api.depends(
+        'holiday_type', 'mode_company_id', 'department_id',
+        'category_id', 'employee_id', 'holiday_status_id',
```

Comment:
missing depends `employee_ids` (with `s`) and `number_of_hours_display`

Reply:
idem (`first level of dependency`)